### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by zackp30**
Outlines how to send a pull request, code standard and issue format.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Contributing code
+
+The fundamental code standard we use
+[StandardJS](http://standardjs.com/rules.html), the rules being
+described in the aforementioned link. We highly recommend you stick to
```
Comment (omnidan) [inline CONTRIBUTING.md]: Actually, we plan on putting linters in the CI process so all PRs that don't stick to the standardjs style get a CI fail.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+this standard to ease the maintenance for us.
+
+1. Fork the repository.
+2. Create a new branch, which very briefly summarizes the change you're going to make.
+
+   For example:
+   
+   - Fixing a typo? ==> `typo`
+   - Documentation addition/fix/etc such as adding the documentation
+     for a method ==> `doc/method-<method name>` or a class ==> `doc/class-<class name>`.
```
Comment (omnidan) [inline CONTRIBUTING.md]: method/class is a java concept and we won't be using it much - maybe use different examples here?

File: CONTRIBUTING.md (hunk 1 above)
Comment (zackp30) [inline CONTRIBUTING.md]: Well, unless they fix the lint errors for us, IMO this should stay. :P

File: CONTRIBUTING.md (hunk 2 above)
Comment (zackp30) [inline CONTRIBUTING.md]: Ironically I'm not that familiar with JS haha. Shall it just be `function` and `prototype`?

File: CONTRIBUTING.md (hunk 1 above)
Comment (omnidan) [inline CONTRIBUTING.md]: I was thinking about adding a line about the linting process.

Comment (zackp30) [inline CONTRIBUTING.md]: Ah, okay.

File: CONTRIBUTING.md (hunk 2 above)
Comment (omnidan) [inline CONTRIBUTING.md]: As a second example you could use `fix-duplicate-events` or something like that. Or maybe add the issue id? Does github have a standard for branch names that reference issues?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+1. Fork the repository.
+2. Create a new branch, which very briefly summarizes the change you're going to make.
+
+   For example:
+   
+   - Fixing a typo? ==> `typo`
+   - Documentation addition/fix/etc such as adding the documentation
+     for a method ==> `doc/method-<method name>`.
+   - Duplicate events? ==> `bug/duplicate-events`.
```
Comment (omnidan) [inline CONTRIBUTING.md]: Fixing a bug? ==> `bug/duplicate-events`.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+   - You could also just use `issue/<issue ID>` as the branch name to refer to a certain issue.
```
Comment (omnidan) [inline CONTRIBUTING.md]: Nice :+1:

**Comment by zackp30** [comment]
Adjusted and squashed.

**Comment by omnidan** [comment]
Thank you :ok_hand:

**Comment by zackp30** [comment]
You're welcome!

**Pull request merged**